src/keyboard_win8.rs

```diff
@@ -157,6 +157,11 @@ pub fn enable(tag: &str, product_code: &str, lang_name: Option<&str>) -> Result<
 
     log::info!("Resetting current active keyboard");
     winuser::set_active_keyboard(original_layout);
+
+    coreglobconfig::sync_language_data();
+    std::thread::sleep_ms(10000);
```

Comment:
Yes, I know this is a long wait, but we have no way to verify when the backup is done and some slow systems might backup slower than others.